### DISCUSSION TITLE
Do you lose starstruck when the repo in question is transfered.

### DISCUSSION THREAD

**Discussion by rickstaa** [Schweinepriester/github-profile-achievements#144] [Q&A] [answered]
I am wondering if it is possible to lose the starstruck achievement when a repository is transferred to an organization :thinking:. Maybe we can test it out using this test account https://github.com/grs-1/starstruck-achievement-repo-transfer-test? Would be great if you could temporarily star it so that I can transfer it to an organization. 🧪If somebody knows the answer already, that would of course also solve my question 🤓.

**Comment by swapnilsparsh** [comment, marked as answer]
No, it will not lost.
There will be a message with an inaccessible link that will be not accessed as you have transferred your repo.
![Start](https://user-images.githubusercontent.com/2644614/179353066-44be6d8a-d00d-4994-9359-59bb0bce5e50.png)

**Reply by rickstaa** [reply]
Great! Thanks for giving me the quickdraw example. I think this is the desired behaviour since when a repo's popularity grows too much, an organization makes more sense.

**Reply by swapnilsparsh** [reply]
Yes, that's the best way.
